.github/label-actions.yml

```diff
@@ -5,13 +5,90 @@
+        tools/dev/msftidy.rb <directory or file>
+        ```
+        
+        It's also possible to automate some of these changes with the `-a` flag:
+        
+        ```
+        rubocop -a <directory or file>
+        ```
+        
+        After making these changes, push to your branch again once all issues have been resolved.
+
+    needs-unique-branch:
+      close: true    
+      comment: |
+        Thanks for your pull request! We require for all contributed code to come from a **from a unique branch** in your repository before it can be merged.
+
+        Please create a new branch in your fork of framework and resubmit this from that branch.
+
+        If you are using Git on the command line that may look like:
+
+        ```
+        # Checkout the master branch
+        git checkout master
+
+        # Create a new branch for your feature
+        git checkout -b <BRANCH_NAME>
+
+        # Add your new files
+        git add modules/my-cool-new-module
+
+        # Commit your changes with a relevant message
+        git commit
+
+        # Push your changes to GitHub
+        git push origin <BRANCH_NAME>
+
+        # Now browse to the following URL and create your pull request!
+        # - https://github.com/rapid7/metasploit-framework/pulls        
+        ```
+
+        This helps protect the process, ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes and allows contributors to make progress while a PR is still being reviewed.
+
+        Please do resubmit from a unique branch, we greatly value your contribution! :tada:
+
+    needs-testing-environment:
+      comment: |
+        Thanks for your pull request! As part of our landing process, we manually verify that all modules work as expected.
+        
+        However in this case we are unable to test this module successfully, either due to software or hardware requirements that we can not replicate.
+        
+        This is different for each pull request, but to help unblock this pull request, please 
+
+        - Comment with links to documentation on how to set up an environment, and provide exact software version numbers to use
+        - Or comment guided steps on how to set up our environment for testing this module
+        - Or send pcaps/screenshots/recordings of it working and logging in as a new user - you can email us msfdev[at]rapid7.com
 
-          - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
-          - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
-          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+        Once there's a clear path for testing and evaluating this module, we can progress with this further.
```

Comment:
I will rename `needs-testing` to `needs-testing-environment` behind the scenes

![image](https://user-images.githubusercontent.com/60357436/79616913-1cb69e00-80fe-11ea-887e-c03449515e3f.png)

Reply:
Small change, I'd suggest the second paragraph ("However in this case...") say instead: 

We have been unable to test this module successfully. This may be due to software or hardware requirements we cannot replicate. 

To help unblock this pull request, please:

etc